experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -493,36 +492,31 @@ describe("SharedTree", () => {
 				};
 				const foo0 = branch.editor.sequenceField({ parent: root0Path, field: fooKey });
 				const foo1 = branch.editor.sequenceField({ parent: root1Path, field: fooKey });
-				branch.editor.setValue(
-					{
-						parent: root0Path,
-						parentField: fooKey,
-						parentIndex: 1,
-					},
-					41,
-				);
-				branch.editor.setValue(
-					{
-						parent: root0Path,
-						parentField: fooKey,
-						parentIndex: 2,
-					},
-					42,
+				rootField.delete(0, 1);
+				rootField.insert(
+					0,
+					singleTextCursor({
+						type: brand("Node"),
+						fields: {
+							foo: [
+								{ type: brand("Number"), value: 0 },
+								{ type: brand("Number"), value: 1 },
+								{ type: brand("Number"), value: 2 },
+							],
+						},
+					}),
```

Comment:
What's the motivation for inserting this bigger subtree instead of just `{ type: brand("Number"), value: 41 }`?